apypie/api.py

```diff
@@ -182,7 +183,7 @@ def _load_apidoc(self):
         return api_doc
 
     def _retrieve_apidoc(self):
-        # type: () -> dict
+        # type: () -> Optional[dict]
```

Comment:
When is this None? This also implies we can write `None` to `apidoc_file`. If we detect `response` is `None`, should we raise an exception?

Reply:
you're right, dropped the `Optional` here.